src/sidebar/services/annotations-exporter.tsx

```diff
@@ -222,21 +222,46 @@ export class AnnotationsExporter {
               return (
                 <article key={annotation.id}>
                   <h2>Annotation {index + 1}:</h2>
-                  <p>
-                    Created at:
-                    <time dateTime={annotation.created}>
-                      {formatDateTime(new Date(annotation.created))}
-                    </time>
-                  </p>
-                  <p>Comment: {annotation.text}</p>
-                  <p>{extractUsername(annotation)}</p>
-                  <p>
-                    Quote: <blockquote>{quote(annotation)}</blockquote>
-                  </p>
-                  {annotation.tags.length > 0 && (
-                    <p>Tags: {annotation.tags.join(', ')}</p>
-                  )}
-                  {page && <p>Page: {page}</p>}
+                  <table>
+                    <tbody>
+                      <tr>
+                        <td>Created at:</td>
+                        <td>
+                          <time dateTime={annotation.created}>
+                            {formatDateTime(new Date(annotation.created))}
+                          </time>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>Author:</td>
+                        <td>{extractUsername(annotation)}</td>
+                      </tr>
+                      {page && (
+                        <tr>
+                          <td>Page:</td>
+                          <td>{page}</td>
+                        </tr>
+                      )}
+                      <tr>
+                        <td>Quote:</td>
+                        <td>
+                          <blockquote style={{ margin: 0 }}>
```

Comment:
If we're removing the margin I think we need to find some other means to convey this is a quote at a glance. Some possibilities:

- Rendering in a lighter shade with a left border like we do in annotations
- Adding quotes around the annotation like text export
- Alternatively maybe some other formatting such as italics?

Reply:
Yep, agree. I'll create an item.